src/sentry/tasks/post_process.py

```diff
@@ -699,12 +713,27 @@ def process_snoozes(job: PostProcessJob) -> None:
                 "user_count": snooze.user_count,
                 "user_window": snooze.user_window,
             }
-            if features.has("organizations:issue-states", group.organization):
-                add_group_to_inbox(group, GroupInboxReason.ONGOING, snooze_details)
-                record_group_history(group, GroupHistoryStatus.ONGOING)
+
+            if features.has("organizations:escalating-issues", group.organization):
+                add_group_to_inbox(group, GroupInboxReason.ESCALATING, snooze_details)
+                record_group_history(group, GroupHistoryStatus.ESCALATING)
+                group.substatus = GroupSubStatus.ESCALATING
+                group.status = GroupStatus.UNRESOLVED
+                group.save(update_fields=["status", "substatus"])
+                analytics.record(
+                    "issue.escalating",
+                    organization_id=group.project.organization.id,
+                    project_id=group.project.id,
+                    group_id=group.id,
+                )
```

Comment:
I think this could call an `update_as_escalating` function with optional parameter `snooze_details`.